lang/lang_en.txt

```diff
@@ -871,6 +877,9 @@
 #
 "Unload"
 
+#
+"Sheet %.7s\x0aZ offset: %+1.3f mm\x0a%cContinue\x0a%cStart from zero"
```

Comment:
Adding this to lang files will generate issues in https://github.com/prusa3d/Prusa-Firmware/pull/2665

Reply:
I've already forgotten if there was a reason for not including this translation